### PR TITLE
Add missing examples.

### PR DESCRIPTION
**Pull request by tjholm**

### CODE REVIEW

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -16,8 +16,8 @@ jobs:
       uses: actions/setup-go@v2-beta
     - name: Install modules
       run: make install-tools
-    - name: Check License Headers
-      run: make license-header-check
+    #- name: Check License Headers
+    #  run: make license-header-check
```
Comment (jyecusch) [inline .github/workflows/test.yaml:20]: Why is the license header check removed in this PR?

File: examples/documents/stream_results.go [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+// [END import]
+
+func stream() {
+	// [START snippet]
+	docs, _ := documents.New()
+
+	query := docs.Collection("Customers").Query()
+
+	// Fetch first page
```
Comment (jyecusch) [inline examples/documents/stream_results.go:33]: comment says fetch first page, but it's getting the stream here. looks like a hangover from the previous snippet

File: examples/documents/stream_results.go [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+	itr, _ := query.Stream()
+
+	for d, err := itr.Next(); err != io.EOF; d, err = itr.Next() {
+		if err != nil {
+			// handle error...
+		}
+
+		fmt.Println("document: ", d)
```
Comment (jyecusch) [inline examples/documents/stream_results.go:41]: Does this need a reference to the value in the string template? e.g.

```go
fmt.Println("document: %v", d)
```

Comment (jyecusch) [reply examples/documents/stream_results.go:41]: If so, the other examples need this change too, if not ignore.

**Review by jyecusch — CHANGES_REQUESTED** [review]

File: .github/workflows/test.yaml (hunk 1 above)
Comment (tjholm) [reply .github/workflows/test.yaml:20]: A left-pad esque issue with add license (doesn't even work with checked out versions that have worked on past commits).
https://github.com/nitrictech/go-sdk/runs/3286678696?check_suite_focus=true

File: examples/documents/stream_results.go (hunk 3 above)
Comment (tjholm) [reply examples/documents/stream_results.go:41]: Println takes (string, ...interface{}), but doesn't contain a formatting component, so it works more like javascript console.log.

File: examples/documents/stream_results.go (hunk 2 above)
Comment (tjholm) [reply examples/documents/stream_results.go]: fixed.

File: .github/workflows/test.yaml (hunk 1 above)
Comment (tjholm) [reply .github/workflows/test.yaml]: sorted it was an issue with a later go version, have cemented it in CI for now.

**Review by jyecusch — APPROVED** [review]

**Pull request merged**